project/Dependencies.scala

```diff
@@ -65,6 +65,7 @@ object Dependencies {
   )
 
   private val sprayServerDependencies = List(
+    "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
```

Comment:
Yeah. tl;dr Khalid was right, it was a c/p error.

Longer version: I flailed around for a long time due to a swagger version mismatch and this little line changed many times.